cypress/component/utils/bucket_utils.spec.ts

```diff
@@ -371,7 +373,7 @@ const dar_collection = {
       dacId: 5,
     },
   ],
-}
+} as unknown as DarCollection
```

Comment:
`as unknown as x` should be avoided since it disables Typescript checking on the object. Consider directly casting the object `as DarCollection`, if this is not possible it means that sub-objects are not properly typed which may indicate a real issue.